public/app/plugins/datasource/azuremonitor/variables.test.ts

```diff
@@ -38,7 +38,7 @@ describe('VariableSupport', () => {
         ],
       } as DataQueryRequest<AzureMonitorQuery>;
       const result = await lastValueFrom(variableSupport.query(mockRequest));
-      expect(result.data[0].source).toEqual(fakeSubscriptions);
```

Comment:
😢 this is referencing a private field -- this only works because everything is typed as `any` and this particular implementation shoves is using ArrayDataFrame